[2023-12-15T18:13:00.744+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data permission to 509
[2023-12-15T18:13:00.748+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data'
[2023-12-15T18:13:00.852+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data permission to 509
[2023-12-15T18:13:00.854+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data'
[2023-12-15T18:13:00.964+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Insert_Data manual__2023-12-15T18:07:58.086087+00:00 [queued]>
[2023-12-15T18:13:01.003+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Insert_Data manual__2023-12-15T18:07:58.086087+00:00 [queued]>
[2023-12-15T18:13:01.005+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-12-15T18:13:01.065+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Insert_Data> on 2023-12-15 18:07:58.086087+00:00
[2023-12-15T18:13:01.077+0000] {standard_task_runner.py:57} INFO - Started process 1787 to run task
[2023-12-15T18:13:01.087+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Insert_Data', 'manual__2023-12-15T18:07:58.086087+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpbo2dbhu_']
[2023-12-15T18:13:01.091+0000] {standard_task_runner.py:85} INFO - Job 86: Subtask Insert_Data
[2023-12-15T18:13:01.224+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data permission to 509
[2023-12-15T18:13:01.226+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=manual__2023-12-15T18:07:58.086087+00:00/task_id=Insert_Data'
[2023-12-15T18:13:01.231+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Insert_Data manual__2023-12-15T18:07:58.086087+00:00 [running]> on host d739bc9bef06
[2023-12-15T18:13:01.416+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Insert_Data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-15T18:07:58.086087+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-15T18:07:58.086087+00:00'
[2023-12-15T18:13:01.435+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/airQuality.py", line 146, in insertValues
    connDB._generate_insert_sql(
TypeError: _generate_insert_sql() missing 2 required positional arguments: 'target_fields' and 'replace'
[2023-12-15T18:13:01.468+0000] {taskinstance.py:211} INFO - Setting task Create_Table to SKIPPED due to dag's `fail_stop` setting.
[2023-12-15T18:13:01.469+0000] {taskinstance.py:211} INFO - Setting task Error_Response to SKIPPED due to dag's `fail_stop` setting.
[2023-12-15T18:13:01.470+0000] {taskinstance.py:211} INFO - Setting task Transform to SKIPPED due to dag's `fail_stop` setting.
[2023-12-15T18:13:01.471+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=air_quality, task_id=Insert_Data, execution_date=20231215T180758, start_date=20231215T181300, end_date=20231215T181301
[2023-12-15T18:13:01.496+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 86 for task Insert_Data (_generate_insert_sql() missing 2 required positional arguments: 'target_fields' and 'replace'; 1787)
[2023-12-15T18:13:01.540+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-15T18:13:01.573+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
